kubevirt-conversion/entrypoint

```diff
@@ -36,8 +36,15 @@ if ls /dev/v2v-disk* 2>/dev/null ; then
   done
 fi
 
+if [ -f "/data/input/conversion.json" ] ; then
+  ln -s /data/input/conversion.json /var/lib/uci/input.json
+fi
+
+echo "Lising content of /var/lib/uci"
+ls -lR /var/lib/uci
```

Comment:
This has to be: `ls -lR /var/lib/uci || true`, otherwise the entrypoint will fail for Kubevirt.